crates/turbopack-tests/Cargo.toml

```diff
@@ -31,3 +31,8 @@ turbopack-env = { path = "../turbopack-env" }
 
 [build-dependencies]
 turbo-tasks-build = { path = "../turbo-tasks-build" }
+
+[features]
```

Comment:
Added this section so `cargo test -p turbopack-tests --features native-tls` is possible. cc @kwonoj